snowfakery/data_generator_runtime.py

```diff
@@ -423,10 +422,7 @@ def __enter__(self, *args):
 
     def __exit__(self, *args):
         for plugin in self.plugin_instances.values():
-            try:
-                plugin.close()
-            except Exception as e:
-                warn(f"Could not close {plugin} because {e}")
+            plugin.close()
```

Comment:
This is a fix that I've wanted to do for a long time and it seemed to help get this PR through faster.

Turning exceptions into warnings is usually a bad idea so let's not do it anymore.